DNN Platform/Library/Entities/Portals/Templates/PortalTemplate.cs

```diff
@@ -1,46 +0,0 @@
-﻿using DotNetNuke.Entities.Portals.Internal;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Xml;
-using System.Xml.Linq;
-
-namespace DotNetNuke.Entities.Portals.Templates
-{
-    public class PortalTemplate : XmlDocument
```

Comment:
Everything else looks good to me, but because this is a public class, we need to mark it deprecated for removal in v11 I believe

Reply:
This was never in the codebase. I added it and then removed it in the same PR.